src/main/java/com/google/cloud/tools/app/impl/cloudsdk/CloudSdkAppEngineFlexibleStaging.java

```diff
@@ -57,6 +58,12 @@ public void stageFlexible(StageFlexibleConfiguration config) throws AppEngineExc
 
     try {
 
+      // Copy docker context to staging
+      if (config.getDockerDirectory() != null && config.getDockerDirectory().exists()) {
```

Comment:
Should we check if the directory contains Dockerfile? Otherwise, it's probably a user error.